spec/graphql/dataloader/async_dataloader_spec.rb

```diff
@@ -1,13 +1,16 @@
 # frozen_string_literal: true
 require "spec_helper"
-
-if Fiber.respond_to?(:scheduler) # Ruby 3+
+if RUBY_VERSION >= "3.1.1"
+  require "async"
   describe GraphQL::Dataloader::AsyncDataloader do
     class AsyncSchema < GraphQL::Schema
       class SleepSource < GraphQL::Dataloader::Source
         def fetch(keys)
           max_sleep = keys.max
-          `sleep #{max_sleep}`
+          # t1 = Time.now
+          # puts "----- SleepSource => #{max_sleep} (from: #{keys})"
```

Comment:
clean up :)

Reply:
I have circled back enough times on this that I think I'd rather just keep it here for next time I need it 😅

Reply:
I have circled back enough times on this that I think I'd rather just keep it here for next time I need it 😅